scripts/main.js

```diff
@@ -1,3 +1,54 @@
+//index.html
+//display the creat app element, when a user click the new app button
+var newAppBtn = document.querySelector("#new-app-btn");
+if (newAppBtn) {
+	newAppBtn.addEventListener("click", function(){
+		document.querySelector("#create-app").style.display = "block";
```

Comment:
As you already use jQuery in the project, you can simplify this code. Instead of 

```js
var newAppBtn = document.querySelector("#new-app-btn");
if (newAppBtn) {
  newAppBtn.addEventListener("click", function(){})
}
```

you can do

```js
$("#new-app-btn").on("click", function () {})
```

It's no problem for jQuery if the button with `id="new-app-btn"` does exist on the page, the rest is simply ignored then